ansible/library/port_alias.py

```diff
@@ -42,30 +42,47 @@
 PORTMAP_FILE = 'port_config.ini'
 ALLOWED_HEADER = ['name', 'lanes', 'alias', 'index', 'speed']
 
+MACHINE_CONF = '/host/machine.conf'
+ONIE_PLATFORM = 'onie_platform'
+ABOOT_PLATFORM = 'aboot_platform'
+
 class SonicPortAliasMap():
     """
     Retrieve SONiC device interface port alias mapping and port speed if they are definded
 
     """
     def __init__(self, hwsku):
-        self.filename = ''
         self.hwsku = hwsku
         return
 
-    def findfile(self):
-        for (rootdir, dirnames, filenames) in os.walk(FILE_PATH, followlinks=True):
-            if self.hwsku == rootdir.split('/')[-1] and len(dirnames) == 0 and PORTMAP_FILE in filenames:
-                self.filename = rootdir + '/' + PORTMAP_FILE
+    def get_platform(self):
+        with open(MACHINE_CONF) as machine_conf:
+            for line in machine_conf:
+                tokens = line.split('=')
+                key = tokens[0].strip()
+                value = tokens[1].strip()
+                if key == ONIE_PLATFORM or key == ABOOT_PLATFORM:
+                    return value
+        return None
+
+    def get_portconfig(self):
```

Comment:
This function returns portconfig file path, rename it to get_portconfig_path? Current name suggests it parses it imo

Reply:
@mykolaf done.